lux/extensions/odm/commands/alembic.py

```diff
@@ -21,6 +21,8 @@ class Command(lux.Command):
                 desc='List available Alembic commands'),
         Setting('msg', ('-m', '--message'), nargs='?', default=None,
                 desc='Message for auto, revision and merge command'),
+        Setting('path', ('-p', '--path'), nargs='?', default=None,
+                desc='Path to migrations directory'),
     )
```

Comment:
we don't need it for that reason, but it seems to be a natural general option
please let me know if we still want to remove it